template/package.json

```diff
@@ -52,27 +52,31 @@
   "devDependencies": {
     "autoprefixer": "^6.3.6",
     "autoprefixer-loader": "^3.2.0",
-    "babel-loader": "^6.2.4",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.4",
     "babel-plugin-react-html-attrs": "^2.0.0",
     "babel-plugin-transform-runtime": "^6.7.5",
     "babel-polyfill": "^6.7.4",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "babel-preset-stage-0": "^6.5.0",
     "chai": "^3.5.0",
     "clean-webpack-plugin": "^0.1.9",
     "cross-env": "^5.1.1",
-    "css-loader": "^0.23.1",
+    "css-loader": "^0.28.11",
     "enzyme": "^2.4.1",
+    "html-webpack-plugin": "^2.30.1",
     "mocha": "^3.0.2",
-    "node-sass": "^3.4.2",
+    "node-sass": "^4.9.0",
     "precss": "^1.4.0",
     "react-addons-test-utils": "^15.0.1",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^4.2.0",
     "sails-hook-autoreload": "^0.13.1",
-    "sass-loader": "^3.2.0",
+    "sass-loader": "^7.0.1",
     "sinon": "^1.17.5",
-    "style-loader": "^0.13.1",
-    "webpack-dev-server": "^1.14.1"
+    "style-loader": "^0.21.0",
+    "webpack": "^3.10.0",
```

Comment:
Is there any reason you're not using `webpack@4`?

Reply:
yes, react hot loader doesn't work properly. It refreshes the page rather ran updating the components.

Reply:
ok, I'll update to latest webpack then